aws/resource_aws_cloudwatch_event_target_test.go

```diff
@@ -648,6 +686,100 @@ CONTAINER_PROPERTIES
 `, rName)
 }
 
+func testAccAWSCloudWatchEventTargetConfigKinesis(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudwatch_event_rule" "cloudwatch_event_rule" {
+  name        = "%[1]s"
+  description = "schedule_batch_test"
+  schedule_expression = "rate(5 minutes)"
+}
+
+resource "aws_cloudwatch_event_target" "test" {
+  arn = "${aws_kinesis_stream.kinesis_stream.arn}"
+  rule = "${aws_cloudwatch_event_rule.cloudwatch_event_rule.id}"
+  role_arn = "${aws_iam_role.iam_role.arn}"
+
+  kinesis_target {
+    partition_key_path = "$.detail"
+  }
+
+  depends_on = [
```

Comment:
The `depends_on` here is unnecessary since we already have explicit dependencies above 👍

Reply:
👍